src/index.html

```diff
@@ -1,13 +1,275 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <nav class="nav">
+          <a href="#menu" class="nav__icon nav__icon--opener"> </a>
+          <a href="/">
+            <img
+            class="logo"
+            src="./images/logo.png"
+            alt="Logo">
+          </a>
+          <a
+            href="tel: +1 234 5555-55-55"
+            class="nav__icon nav__icon--phone"
+          >
+          </a>
+        </nav>
+        <h1 class="header__main-title">
+          Miami
+          <br>
+          Condo Kings
+        </h1>
+      </div>
+    </header>
+    <div class="menu" id="menu">
+      <div class="container menu__container">
+        <nav class="nav" >
+          <a href="#" >
+            <img
+            class="nav__icon"
+            src="./images/cross.svg"
+            alt="cross"
+            >
+          </a>
+          <a href="/" >
```

Comment:
```suggestion
          <a href="#" >
```
Обычно выбирают решетку в качестве дефолтного пути